guides/_subsections/download-an-automated-rule.md

```diff
@@ -1,17 +1,17 @@
 ## [Download an Automated Rule](#download-an-automated-rule)
 
-Automated Rules can easily be downloaded for viewing and editing.
+`Automated Rules` can easily be downloaded for viewing and editing.
 
 
 <ol>
   <li>
       {% include howto_step.html
-        summary="Open action menu for a selected rule"
+        summary="Open Action Menu for a Selected Rule"
         image-name="2.3.0/download-an-automated-rule-1.png"
         caption="Click the action overflow three-dot menu on the right side of selected rule entry in the table."
```

Comment:
Click the action overflow \"&#65049;\" <i>three-dot</i> menu on the right side...